lib/tasksNames.ts

```diff
@@ -14,6 +14,9 @@ export const MULTISIG_IS_PROPOSAL_TIMELOCK_REACHED =
 export const MULTISIG_IS_OWNER = "stakedCelo:multiSig:isOwner";
 export const MULTISIG_IS_CONFIRMED_BY = "stakedCelo:multiSig:isConfirmedBy";
 export const MULTISIG_ENCODE_PROPOSAL_PAYLOAD = "stakedCelo:multiSig:encode:proposal:payload";
+export const MULTISIG_SUBMIT_PROPOSAL_SET_DEPENDENCIES =
```

Comment:
Update the name to match actual task. This only encodes the proposal for setting dependencies without submitting.